kartograf/rpki/fetch.py

```diff
@@ -58,21 +72,24 @@ def fetch_rpki_db(context):
     # Download TALs and presist them in the RPKI data folder
```

Comment:
nit: Just noticed this (unrelated) typo, you can fix that if you make further changes here, thanks!

```suggestion
    # Download TALs and persist them in the RPKI data folder
```